src/SignalR/server/Core/src/HubInvocationContext.cs

```diff
@@ -2,20 +2,41 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System.Collections.Generic;
+using Microsoft.AspNetCore.Authorization;
 
 namespace Microsoft.AspNetCore.SignalR
 {
+    /// <summary>
+    /// Context for a Hub invocation.
```

Comment:
super nit: You capitalize Hub here but have it lower everywhere else.

Reply:
:+1: capitalizing Hub is probably correct